include/sir/config.h

```diff
@@ -69,10 +69,33 @@
 # endif
 
 /**
- * Define to use <CR><LF> line endings, otherwise use <LF>.
+ * Define the system carriage return (CR) character.
  */
-# if !defined(SIR_EOL_CRLF)
-#  undef SIR_EOL_CRLF
+# if !defined(SIR_EOL_CR)
+#  define SIR_EOL_CR "\r"
+# endif
+
+/**
+ * Define the system line feed (LF) character.
```

Comment:
`The line feed (LF) character to use in the end of line sequence.`

Reply:
Done.